pkg/utils/globalTypes/health/health.go

```diff
@@ -38,10 +39,24 @@ func NewHealthInfo(parameters *config.ConfigParameters) *HealthInfo {
 		MaxDayCritical: parameters.MaxDayCritical,
 		HPLossBase:     parameters.HPLossBase,
 		HPLossSlope:    parameters.HPLossSlope,
+		maxPlatFood:    parameters.FoodOnPlatform,
 	}
 }
 
+// This function takes in an initial HP and a goal HP, and returns how much food you should eat to
+// achieve the goal HP, while taking into account HP decay.
+// This function should be called only for HP values above WeakLevel.
```

Comment:
> This function should be called only for HP values above WeakLevel.
What's the behavior if it's called on that? Undefined or breaks? Could add an error to the function (if it's not too hard of course)

Reply:
Gonna say it's undefined - because for critical HP the behaviour of an agent's HP is defined on the number of days they spend in critical